server/game/cards/characters/02/sergregorclegane.js

```diff
@@ -0,0 +1,40 @@
+const DrawCard = require('../../../drawcard.js');
+
+class SerGregorClegane extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onPillage: (event, challenge, card, discarded) => card === this && discarded.getType() === 'character'
+            },
+            handler: context => {
+                var discarded = context.event.params[3];
+                discarded.controller.moveCard(discarded, 'dead pile');
+                this.game.addMessage('{0} uses {1} to place {2} in {3}\'s dead pile.', this.controller, this, discarded, discarded.controller);
```

Comment:
This is super pedantic but I don't think we typically end sentences in chat with '.'s. We should be consistent either way